cif_pow.dic

```diff
@@ -7418,6 +7418,75 @@ save_PD_INSTR_DETECTOR
     _name.object_id               PD_INSTR_DETECTOR
     _category_key.name            '_pd_instr_detector.id'
 
+    loop_
+      _description_example.case
+      _description_example.detail
+;
+        loop_
+        _pd_instr_detector.id
+        _pd_instr.dist_spec_detc
+        _pd_instr.slit_ax_spec_detc
+        _pd_instr.slit_eq_spec_detc
+        _pd_instr_detector.instr_id
+            fd343493   320   8.5   0.075   b4131be5
+            47540661   320   8.5   0.075   b4131be5
+;
+;
+        These detectors are paired with the instrument identified by the
+        id b4131be5.
+
+        There are two detectors, fd343493 and 47540661. Both detectors are
+        positioned 320 mm from the specimen, and are defined by two slits,
+        8.5 mm in the axial direction, and 0.075 mm in the equatorial direction.
+        As the instrument description says that this instrument has 'Dual strip
+        detectors', it can be assumed that the detector window is 8.5 mm wide,
+        and each strip, or pixel, in the detector is 75 μm high.
+;
+;
+        _pd_instr_detector.id             WhizzBang
+        _pd_instr_detector.instr_id       58b6d83b
+        _pd_instr.2theta_monochr_post      31.0
+        _pd_instr.dist_anal_detc           47
+        _pd_instr.dist_spec_anal          135
+        _pd_instr.monochr_post_spec       'Graphite mosaic monochromator'
+        _pd_instr.slit_ax_spec_anal         9.5
+        _pd_instr.slit_eq_spec_anal         0.5
+        _pd_instr.soller_eq_spec_anal       2.5
+;
+;
+        The 'WhizzBang' detector is paired with the instrument identified by the
+        id 58b6d83b.
+
+        The post-specimen, graphite, mosaic monochromator is set at 31.0° 2θ.
+        The monochromator is situated 47 mm from the detector. The distance from
+        the specimen to the monochromator is 135 mm. The width of the beam is
+        defined at the monochromator by a 9.5 mm mask. There is a 0.5 mm
+        equatorial slit between the specimen and monochromator, which can be
+        assumed to be on the detector circle, and acts as the virtual detector.
```

Comment:
I worry about assumptions. As a programmer, how do I know when to assume? Are we missing some data names that would indicate where the slit is?

Reply:
We are missing many data names for describing an instrument. #157 refers.

Reply:
Shall we flag that it is currently not possible to specify the location of the slit, so that information should be provided in `_pd_instr.special_details`?

Reply:
Yes. Should also go into VolG that PD_INSTR(_DETECTOR) is not adequate to completely describe an instrument, and so please use special_details.

Reply:
> There exists _pd_instr.special_details, but not _pd_instr_detector.special_details. Given that the two categories are supposed to be used together, I don't think this is a real issue.

Agreed